jenkins/python-setup.sh

```diff
@@ -24,7 +24,8 @@ while read -r _pdir; do
     fi
 done <<< "$(ls -1d ${_prefix}/lib*/python3.*/site-packages 2> /dev/null)
 $(head -n 1 ${_prefix}/lib*/python3.*/site-packages/pbench.egg-link 2> /dev/null)"
+
+# We need the path to the source tree server library code since we don't
+# install it directly for tests.
 PYTHONPATH="${progdir}/server/lib":"${progdir}/lib":$PYTHONPATH
 export PYTHONPATH
-
-python3 setup.py develop --prefix="${_prefix}"
```

Comment:
I'm curious about your removal of the `setup.py` invocation.  Does this really not accomplish anything that we need?

I think that it is required for the Click-based commands, but I guess those don't exist in 0.69...we don't need it for anything else?

What prompted you to remove it?

Reply:
I prompted here.  We met and reviewed it and tried it out, and it does literally nothing.  Try it and see.

Reply:
I _think_ it's redundant (i.e., when we're using `tox`) because we're specifying [`usedevelop`](https://tox.readthedocs.io/en/latest/config.html#conf-usedevelop) in the `tox.ini` file.  (There are _so many_ knobs and settings, I cannot keep track of them all.)

Unfortunately, we use this stuff in several different ways, and I'm not aware of the nuances and requirements of them all, so, under the assumption that someone put this invocation in here for a good reason, I felt obliged to question its removal.  🤕  Nevertheless, I'm a huge fan of deleting things that we don't need, so...if we really don't need it, that's great!

(Hey, @portante, while you're here...I think you've got a few open conversations which you can mark as resolved, now. 😉)

Reply:
So, it's not _only_ redundant, but it is useless _as well_?  Superb!  😁